prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -382,4 +412,17 @@ export class CollegeCertificationFormComponent implements OnInit {
   private filterPractices(collegeCode: number): PracticeConfig[] {
     return this.practices.filter(p => p.collegePractices.map(cl => cl.collegeCode).includes(collegeCode));
   }
+
+
+  private checkLicenseIfDiscontinued() {
+    if (this.collegeCode.value && this.licenseCode.value) {
+      this.licenseClassDiscontinued = this.isCertificationDiscontinued(this.collegeCode.value, this.licenseCode.value);
+      this.collegeDiscontinued = this.colleges.some(c => c.code == this.collegeCode.value);
+    }
+  }
+
+  private isCertificationDiscontinued(collegeCode: number, licenseCode: number): boolean {
+    let collegeLicense = this.configService.licenses.find(l => l.code === licenseCode);
```

Comment:
Rather than `collegeLicense`, should that just be `license`?

Reply:
It is updated.